Packet++/src/TelnetLayer.cpp

```diff
@@ -0,0 +1,315 @@
+#define LOG_MODULE PacketLogModuleTelnetLayer
+
+#include "Logger.h"
+#include "TelnetLayer.h"
+
+#include "GeneralUtils.h"
+#include "SystemUtils.h"
+
+#include <string.h>
+
+namespace pcpp
+{
+
+    void TelnetLayer::parseTelnetFields()
+    {
+        // Since only contains data return immediately
+        if ((m_Data[0] != 255) || (m_Data[0] == 255 && m_Data[1] == 255))
+        {
+            isData = true;
+            return;
+        }
```

Comment:
I've seen cases where the same packet contains both commands and data, here is one example:
[telnet-commands-and-data.zip](https://github.com/seladb/PcapPlusPlus/files/8152387/telnet-commands-and-data.zip)